lib/apiservers/engine/backends/container.go

```diff
@@ -165,14 +164,13 @@ func (c *Container) ContainerCreate(config types.ContainerCreateConfig) (types.C
 		config.Config.Entrypoint = layer.Config.Entrypoint
 	}
 	config.Config.Env = append(config.Config.Env, layer.Config.Env...)
-
 	log.Printf("config.Config' = %+v", config.Config)
 
 	// Call the Exec port layer to create the container
-	host, err := guest.UUID()
+	host, err := os.Hostname()
```

Comment:
it is fixed for some reason the PR is not updating... 

please look here.

https://github.com/matthewavery/vic/blob/docker-rm/lib/apiservers/engine/backends/container.go#L509